.github/workflows/_netlify.yml

```diff
@@ -29,11 +29,11 @@ on:
       NEXT_PUBLIC_UNLOCK_ENV:
         required: false
       # unlock-app only
-      BASE64_WEDLOCKS_PUBLIC_KEY:
+      NEXT_PUBLIC_BASE64_WEDLOCKS_PUBLIC_KEY:
         required: false
```

Comment:
What if we make required to true here?

Reply:
wouldn't work because we reuse this action for multiple apps. 

So we can't make any of the variable required unless it's needed for all.